src/main/java/com/aws/greengrass/util/Utils.java

```diff
@@ -212,7 +214,7 @@ public static Path homePath(String s) {
     public static String extension(String s) {
         if (s != null) {
             int dp = s.lastIndexOf('.');
-            if (dp > s.lastIndexOf('/')) {
+            if (dp > s.lastIndexOf(File.separatorChar)) {
```

Comment:
not necessarily true, you can use `/` on windows too. We should account for either

Reply:
`DefaultFileSystem.getFileSystem().normalize(s)` will ensure the path is properly formatted for the given filesystem

Reply:
DefaultFileSystem is not public. Since we already depend on Apache commons, switching to FilenameUtils.getExtension which works for windows